### PR TITLE
Feature/integrate apps into createapp

### PR DESCRIPTION
**Pull request by liayoo**
- Set apps/<appName> owner and rule configs in _createApp
- Check shardOwner permissions in setUpDbForSharding(), try creating an app if it hasn't been created yet
- Updated test cases & removed legacy setup code

### CODE REVIEW

File: db/functions.js [hunk 1]
```diff
@@ -577,6 +589,24 @@ class Functions {
     }
     if (adminConfig) {
       sanitizedVal[PredefinedDbPaths.MANAGE_APP_CONFIG_ADMIN] = adminConfig;
+      const appPath = PathUtil.getAppPath(appName);
+      const owner = {
+        [OwnerProperties.OWNER]: {
+          [OwnerProperties.OWNERS]: {
+            [OwnerProperties.ANYONE]: buildOwnerPermissions(false, false, false, false),
```
Comment (platfowner) [inline db/functions.js:596]: This config for anyone (*) can be just skipped as there is no permission here?

File: p2p/server.js [hunk 2]
```diff
@@ -615,16 +617,50 @@ class P2pServer {
     const parentChainEndpoint = GenesisSharding[ShardingProperties.PARENT_CHAIN_POC] + '/json-rpc';
     const ownerPrivateKey = ChainUtil.getJsObject(
         GenesisAccounts, [AccountProperties.OWNER, AccountProperties.PRIVATE_KEY]);
+    const shardOwner = GenesisSharding[ShardingProperties.SHARD_OWNER];
+    const shardingPath = GenesisSharding[ShardingProperties.SHARDING_PATH];
+    const appName = _.get(ChainUtil.parsePath(shardingPath), 1);
```
Comment (platfowner) [inline p2p/server.js:622]: minor: Can the appName's validity be checked for mis-configuration cases? e.g.
```
const appName = _.get(ChainUtil.parsePath(shardingPath), 1, null);
if (!appName) {
  ... // Handle this case
}
```

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Great to see this change, thanks!

**Comment by liayoo** [comment]
Thanks for the review! Now merging..

**Pull request merged**